lib/src/chip/Chip.stories.tsx

```diff
@@ -50,33 +44,56 @@ const opinionatedTheme = {
 export const Chromatic = () => (
   <>
     <ExampleContainer>
-      <Title title="Basic chip" theme="light" level={4} />
-      <DxcChip label="Default Chip" />
```

Comment:
Maybe keep the default one with neither suffix or prefix?

Reply:
Done!